### PR TITLE
Fixed my typo with revolvers

### PR DESCRIPTION
**Pull request by Sabess**
Unfortunately missed one typo in testing, which broke a certain optional sprite for the revolver. Oops. Also, I changed the default name of the revolver to ".45 revolver" after realising it could potentially be confused with the antag revolver when being aimed or etc.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 1]
```diff
@@ -67,7 +67,7 @@
 		return 1
 
 /obj/item/weapon/gun/projectile/revolver/detective45
-	name = "revolver"
+	name = ".45 revolver"
```
Comment (Belsima) [inline code/modules/projectiles/guns/projectile/revolver.dm]: Honestly, I prefer the ambiguity, but I have dumb opinions.

Comment (Belsima) [reply code/modules/projectiles/guns/projectile/revolver.dm]: I like the description idea better.

**Comment by Anewbe** [comment]
In the name is fine, I'm pretty sure you can rename them anyway.

File: code/modules/projectiles/guns/projectile/revolver.dm (hunk 1 above)
Comment (Sabess) [reply code/modules/projectiles/guns/projectile/revolver.dm]: the description is already different, and yeah you could just name it "revolver" if you really wanted

**Pull request merged**